### DISCUSSION TITLE
Welcome to Hajk Discussions!

### DISCUSSION THREAD

**Discussion by jacobwod** [hajkmap/Hajk#782] [General]
## 👋 Welcome to Hajk Discussions!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Hallbergs** [comment]
Great initiative @jacobwod!

My name is Henrik, and i currently work as a developer at the property management department in Gothenburg. I've been coding hajk for 2-ish years, and i love the project!

I'm looking forward to many exciting discussions on future plans for Hajk!

**Comment by jacobwod** [comment]
I noticed I haven't written a word about myself yet. Here it comes: I work as a developer for a department within Halmstad that's responsible for GIS-related topics within the municipality. I started my journey with Hajk at the end of 2017 when we decided to do a complete overhaul of the Client application, and has been working on the project ever since, fixing bugs, updating existing features and adding new functionality to the different parts of the Hajk solution.
